prow/github/fakegithub/fakegithub.go

```diff
@@ -76,6 +76,9 @@ type FakeClient struct {
 	// Fake remote git storage. File name are keys
 	// and values map SHA to content
 	RemoteFiles map[string]map[string]string
+
+	// org/repo/ref
```

Comment:
Could we improve this doc, please?

Reply:
Could also use an anonymous struct here in case not flattening it to a string would be clearer

Reply:
Done